docs(gpu): understanding nvidia fp8 format (#2303)
scaleway/docs-content: compute/gpu/reference-content/understanding-nvidia-fp8.mdx

---
meta:
  title: Understanding the NVIDIA FP8 format
  description: This section provides information about NVIDIA FP8 (8-bit floating point) format
content:
  h1: Understanding the NVIDIA FP8 format
  paragraph: This section provides information about NVIDIA FP8 (8-bit floating point) format
tags: NVIDIA FP8 GPU cloud
dates:
  validation: 2023-10-23
  posted: 2023-10-23
categories:
  - compute
---

> With the release of the H100 GPU, NVIDIA introduced support for a new datatype called FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions.

Reviewer: ```suggestion
With the release of the H100 GPU, NVIDIA introduced support for a new datatype called FP8 (8-bit floating point), enabling higher throughput of matrix multipliers and convolutions.
```

Reviewer: Not sure if this should be multipliers or multiples?

> FP8, an 8-bit floating point standard, was jointly developed by NVIDIA, ARM, and Intel to speed up AI development by improving memory efficiency during AI training and inference processes.

Reviewer 2: ```suggestion
FP8 is an 8-bit floating point standard which was jointly developed by NVIDIA, ARM, and Intel to speed up AI development by improving memory efficiency during AI training and inference processes.
```

The ongoing industry evolution is evident in the shift from 32-bit to 16-bit, and currently, to 8-bit precision formats. This change is especially advantageous for transformer networks, crucial AI breakthroughs, as they perform better with the detailed precision provided by 8-bit floating point formats.

This uniform standard ensures compatibility across diverse hardware and software environments, thus driving computing capabilities to unprecedented levels.

> FP8 sets forth two distinct eight-bit floating-point formats, E5M2 and E4M3, to facilitate interoperability between different hardware platforms. It aligns closely with the current IEEE 754 standards for floating-point computations, achieving an ideal harmony between hardware efficiency and software performance. This strategy seeks to leverage existing frameworks, quicken uptake, and boost developer efficiency.

Reviewer 2: ```suggestion
FP8 sets forth two distinct eight-bit floating-point formats, E5M2 and E4M3, to facilitate interoperability between different hardware platforms. It aligns closely with the current IEEE 754 standards for floating-point computations, achieving an ideal harmony between hardware efficiency and software performance. This strategy seeks to leverage existing frameworks, speed up uptake, and boost developer efficiency.
```
